### PR TITLE
Add option to set GRUB boot timeout

### PR DESCRIPTION
**Pull request by MEMESCOEP**
I've added the option to change the GRUB boot timeout because I think it could be useful during debugging. If the value isn't set it will default to 0 seconds.

### CODE REVIEW

**Review by quajak — CHANGES_REQUESTED** [review]
Thanks for the contribution! Looks good except one small nitpick.

**Comment by MEMESCOEP** [comment]
What needs to be changed?

File: source/Cosmos.Build.Tasks/build/Cosmos.Build.targets [hunk 1]
```diff
@@ -383,10 +383,12 @@
         <CreateGrubConfig TargetDirectory="$(IntermediateIsoDirectory)"
                               BinName="$([System.IO.Path]::GetFileName('$(BinGzFile)'))"
                               Modules="gzio"
+                              timeout="$(Timeout)"
```
Comment (quajak) [inline source/Cosmos.Build.Tasks/build/Cosmos.Build.targets:386]: Just to confirm it ok not to capitalize the timeout in the call here?

Comment (MEMESCOEP) [reply source/Cosmos.Build.Tasks/build/Cosmos.Build.targets:386]: Wait I did capitalize that in my code? I'll check it out and see what's going on

Comment (MEMESCOEP) [reply source/Cosmos.Build.Tasks/build/Cosmos.Build.targets]: Problem has been fixed

**Review by quajak — APPROVED** [review]
Thank you!

**Pull request merged**